BLOGPOST.md

```diff
@@ -235,7 +233,7 @@ Also, the [Introducing Componentize-Py: A Tool for Packaging Python Apps as Comp
 
 ### IPFS
 
-Homestar and Everywhere Computer currently use [IPFS Kubo][kubo] as a storage layer. Before we start into the next section, [install IPFS Kubo][install-ipfs] and start the IPFS daemon:
+Homestar and Everywhere Computer currently use [IPFS Kubo][kubo] as a storage layer for Wasm modules. In the near future, we'll support other forms of distributed storage. Before we start into the next section, [install IPFS Kubo][install-ipfs] and start the IPFS daemon:
```

Comment:
```suggestion
Homestar and Everywhere Computer currently use [IPFS Kubo][kubo] as a storage layer for Wasm components. In the near future, we'll support other forms of distributed storage. 

Before we start into the next section, [install IPFS Kubo][install-ipfs] and start the IPFS daemon:
```
"components" to align with our earlier usage.

Also, new paragraph to emphasize the next action.